src/SignalR/clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -1019,29 +1019,6 @@ describe("hubConnection", () => {
         }
     });
 
-    it("populates the Content-Type header when sending XMLHttpRequest", async () => {
```

Comment:
Why did we need to delete this test? I thought this should still work for long polling sends with a JSON payload.

Reply:
The send http requests don't affect the headers visible from the Hub